resources/metadata/fortify.yaml

```diff
@@ -32,435 +32,419 @@ spec:
+        - PARAMETERS
+        - STAGES
+        - STEPS
     - name: autodetectClasspath
       type: bool
-      description: Whether the classpath is automatically determined via build tool i.e. maven or pip or not at all
+      description: "Whether the classpath is automatically determined via build tool i.e. maven or pip or not at all"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
       default: true
     - name: mustAuditIssueGroups
       type: string
-      description: Comma separated list of issue groups that must be audited completely
+      description: "Comma separated list of issue groups that must be audited completely"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
-      default: 'Corporate Security Requirements, Audit All'
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: "Corporate Security Requirements, Audit All"
     - name: spotAuditIssueGroups
       type: string
-      description: 'Comma separated list of issue groups that are spot checked and for which `spotCheckMinimum` audited issues are enforced'
+      description: "Comma separated list of issue groups that are spot checked and for which `spotCheckMinimum`
+        audited issues are enforced"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
-      default: 'Spot Checks of Each Category'
+        - PARAMETERS
+        - STAGE
+        - STEPS
+      default: "Spot Checks of Each Category"
     - name: pythonRequirementsInstallSuffix
       type: string
-      description: 'The suffix for the command used to install the requirements file in `buildTool: ''pip''` to populate
-        the build environment with the necessary dependencies'
+      description: "The suffix for the command used to install the requirements file in `buildTool: 'pip'` to populate
+        the build environment with the necessary dependencies"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
     - name: pythonVersion
       type: string
-      description: 'Python version to be used in `buildTool: ''pip''`'
+      description: "Python version to be used in `buildTool: 'pip'`"
       scope:
-      - GENERAL
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - GENERAL
+        - PARAMETERS
+        - STAGES
+        - STEPS
       default: python3
     - name: uploadResults
       type: bool
-      description: Whether results shall be uploaded or not
+      description: "Whether results shall be uploaded or not"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
       default: true
     - name: buildDescriptorFile
       type: string
-      description: 'Path to the build descriptor file addressing the module/folder
+      description: "Path to the build descriptor file addressing the module/folder
```

Comment:
Is that somehow related to `modulePath`?

Reply:
Definitely related. If you are asking whether it is somehow redundant, I don't know without looking more deeply. I know modulePath is prepended to paths in various places, it is definitely not the path to the build descriptor or the name of a module. I need to look more closely whether buildDescriptorFile is relative to modulePath.

Reply:
I think for this PR we can ignore that